src/Backend/OnboardingInsuranceAPI/Areas/Insurance/PutInsurance.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using OnboardingInsuranceAPI.Services;
+
+namespace OnboardingInsuranceAPI.Areas.Insurance;
+
+public class PutInsurance: IHandler
+{
+    private readonly DataContext _context;
+
+    public PutInsurance(DataContext context)
+    {
+        _context = context; 
+    }
+
+    public async Task UpdateInsurance(InsuranceData item)
+    {
+        var insurance = await _context.Users.FirstOrDefaultAsync(i => i.Pid == item.Pid);
+
+        insurance.CarInsurance = item.CarInsurance;
+        _context.Update(insurance);
+        await _context.SaveChangesAsync();
+        Console.WriteLine("Information saved...");
```

Comment:
This is pointless here. Because when deployed you can never get to this information, and when you use it locally you can just put this into Debug Log or Trace Log.

Reply:
Also you are not providing any useful information.

Reply:
Ok, Yeah I'll delete Console.WriteLine("Information saved...");